federatedgroups/lib/MixedGroupShareProvider.php

```diff
@@ -520,5 +598,64 @@ private function updateShareAttributes(\OCP\Share\IShare $share, $data) {
 		return $share;
 	}
 
+	public function getShareById($id, $recipient = null) {
```

Comment:
I don't see how this function is different from the parent function. Except you changed the parameter name from `recipientId` to `recipient` which will result in error.

Reply:
I realize now that this is needed to override the createShare function. Makes sense to me. Only the function parameter should be fixed then.